cdk/settings.py

```diff
@@ -83,10 +88,12 @@ class StackSettings(BaseSettings):
     # Cluster scaling max
     BATCH_MAX_VCPU: int = 10
 
+    MAX_ACTIVE_JOBS: int = 10_000
```

Comment:
No change for this PR, but I'm curious if you're planning on having separate AWS Batch compute environments for Landsat vs Sentinel-2 or for auxdata processing?